### ISSUE TITLE
The SDK 'Microsoft.NET.Sdk.Web' specified could not be found.

### ISSUE THREAD

**Issue by markcaldropoli** [dotnet/docfx#7523]
**Operating System**: `Windows`
**DocFX Version Used**:  `2.58.0`
**Template used**: `default`
**Steps to Reproduce**:

I have a solution with 2 projects in NET Core 3.1:
- One uses Microsoft.NET.Sdk and the documentation properly loads
- The other uses Microsoft.NET.Sdk.Web and it results in:  
`The SDK 'Microsoft.NET.Sdk.Web' specified could not be found.`

**Expected Behavior**:

Documentation loads for both projects

**Actual Behavior**:

`The SDK 'Microsoft.NET.Sdk.Web' specified could not be found.`

As a result of this, the documentation is unable to load for the project using the Web Sdk. The documentation is able to load for the regular Sdk project when serving.

**Comment by jaderanderson** [comment]
I have precisely the same issue, except I'm using NET Core 5 and version 2.58.9. I tried to run on a single project (the API one we need to document) and get the same error.

Tried creating environment variables according to other issues reported but nothing works. I have net core 5.0 sdk installed AND visual studio 2019 enterprise.

The project builds just fine, using docfx externally or as a nuget on said project yielded no results.
